homebrew.rb

```diff
@@ -0,0 +1,124 @@
+# Homebrew
+# -----
+
+require "rake/packagetask"
+
+$LOAD_PATH.unshift(File.expand_path("../lib", __FILE__))
+
+GEMS_DIR = "vendor/gems"
+GH_PAGES_DIR = "gh-pages"
+HOMEBREW_FORMULAE_DIR = "homebrew-formulae"
+
+namespace :gems do
+  desc "Vendorize dependencies"
+  task :vendorize do
+    system("vendor/vendorize", GEMS_DIR)
+  end
+
+  desc "Remove vendorized dependencies"
+  task :clean do
+    FileUtils.rm_r(GEMS_DIR) if Dir.exist?(GEMS_DIR)
+  end
+end
+
+desc "Push new release"
+task homebrewrelease: ["release:build", "release:push", "release:clean"]
+
+namespace :release do
+  desc "Build a new release"
+  task build: ["tarball:build", "homebrew:build"]
+
+  desc "Push sub-repositories"
+  task push: ["tarball:push", "homebrew:push"]
+
+  desc "Clean all build artifacts"
+  task clean: ["gems:clean", "tarball:clean", "homebrew:clean"]
+end
+
+namespace :homebrew do
+  desc "Generate homebrew formula and add it to the repo"
+  task build: ["checkout", "formula:build", "commit"]
+
+  desc "Checkout homebrew repo locally"
+  task :checkout do
+    `git clone https://github.com/CocoaPods/homebrew.git #{ HOMEBREW_FORMULAE_DIR }`
```

Comment:
No, since we would need a fork of it to submit new PRs. I'm waiting to fork it until it's good to go.